src/linter/linterReporting.ts

```diff
@@ -0,0 +1,44 @@
+export const MESSAGES = {
+	SHORT__NO_DIRECT_DATATYPE_ACCESS: "Deprecated access to DataType ('{0}')",
```

Comment:
IMHO the messages are not very consistent:

`Deprecated access to DataType ('{0}')`
`Deprecated access to enum pseudo module '{0}'`

- Both messages are regarding the pseudo module usage, but the DataType doesn't mention it.
- DataType uses brackets, which isn't widely used in the other messages.

```suggestion
	SHORT__NO_DIRECT_DATATYPE_ACCESS: "Deprecated access to DataType pseudo module '{0}'",
```

Reply:
Done